File: APPTOPI/app/src/main/java/io/topi/apptopi/service/GitServiceRetrofit.java
package io.topi.apptopi.service;

import com.google.gson.JsonObject;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Headers;
import retrofit2.http.Query;

public interface GitServiceRetrofit {
    @Headers("ContentType: application/json")

    @GET("repositories")
    Call<JsonObject> getList(@Query("q") String query, @Query("sort") String sort, @Query("page") String page);
}

File: APPTOPI/app/src/main/java/io/topi/apptopi/main/repositorio/GitServiceRetrofit.kt
package io.topi.apptopi.main.repositorio

import com.google.gson.JsonObject
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Headers
import retrofit2.http.Query

interface GitServiceRetrofit {
    @Headers("ContentType: application/json")

    @GET("repositories")
    fun getList(@Query("q") query: String, @Query("sort") sort: String, @Query("page") page: String): Call<JsonObject>
}
